FirebaseAdmin/FirebaseAdmin/Auth/FirebaseUserManager.cs

```diff
@@ -195,5 +248,36 @@ private TResult SafeDeserialize<TResult>(string json)
                 throw new FirebaseException("Error while calling Firebase Auth service", e);
             }
         }
+
+        private class UserQuery
```

Comment:
Would be nice to have a top-level comment describing this class.

Reply:
Done